src/python/pants/option/global_options.py

```diff
@@ -194,8 +195,12 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.codegen.grpcio.python',
                       'pants.backend.codegen.wire.java',
                       'pants.backend.project_info'],
-             help='Load backends from these packages that are already on the path. '
+             help='Load v1 backends from these packages that are already on the path. '
                   'Add contrib and custom backends to this list.')
+    register('--backend-packages2', advanced=True, type=list,
+             default=[],
+             help='Load v2 backends from these packages, which must already be on the PYTHONPATH,'
```

Comment:
All of the plugin and backend options could stand some massively improved help strings.

My understanding of the reason why `--plugins2` is necessary is that adding something to the plugin list causes it to be resolved AND then loaded as a backend. But that should probably be explained here, or on the docsite (with a link from here to there).

But also, it's not clear to me what the semantics of putting a backend in both of the `--backend-packages` lists would be. Is that the correct way to load both the v1 and v2 features of a backend? Or does v1 imply v2, etc?

Reply:
That is the correct way to load both. v1 does not imply v2. Will clarify comments.

Reply:
> v1 does not imply v2

Interesting. What was the thinking there? It's possible that v1 should imply v2, as some v1 backends might synchronously consume the Scheduler and want to use `@rules` to do it. For example, we've discussed (but not started) porting the `jvm_compile` task to v2, and doing that would involve v1 consuming rules.

Reply:
The thinking was to keep things simple. So if you need both, enumerate them in both places I guess? That more explicitly models what's going on.